docs/sphinx/examples/cpp/other/builder/builder.cpp

```diff
@@ -17,6 +17,14 @@
 #include <cudaq/gradients.h>
 #include <cudaq/optimizers.h>
 
+static bool results_are_close(const cudaq::sample_result &f1,
```

Comment:
Am I missing something or is this returning true even if f1 =/= f2 ?

Reply:
This is completely fake. The kernel has no quantum code and hence does nothing. So this stub doesn't actually compare anything. As a stub, it just rubber stamps the results.